### ISSUE TITLE
MMPM does not detect git failures

### ISSUE THREAD

**Issue by accountnumber3** [Bee-Mar/mmpm#10]
```$ mmpm -i MMM-Dad-Jokes
Installing MMM-Dad-Jokes @ /mnt/dev/MagicMirror/modules/MMM-Dad-Jokes

Cloning repository for MMM-Dad-Jokes...

Cloning into '/mnt/dev/MagicMirror/modules/MMM-Dad-Jokes'...
fatal: cannot copy '/usr/share/git-core/templates/description' to '/mnt/dev/MagicMirror/modules/MMM-Dad-Jokes/.git/description': Operation not permitted

Repository cloned.
```

Pretty sure I've messed up my git-core somehow. MMPM does not detect that the clone failed.

**Comment by accountnumber3** [comment]
I solved the git issues. My MM install is on an NFS share from freenas. At some point in the past I corrupted the file permissions when I changed from a Windows dataset to a UNIX dataset. I fixed it by setting it from UNIX to Windows, then Windows to UNIX, and doing a recursive permission change.

Obviously that is wayyyyy outside the scope of MMPM, but it should at least be able to detect the failure.
